checkers/Board.py

```diff
@@ -68,15 +69,31 @@ def find_valid_moves(self, player_turn):
 
         # check to make sure between boundaries
         if left_grid_row >= 0 and left_grid_row <= 7 and left_grid_col >= 0 and left_grid_col <= 7:
-            # if it is empty
+
+            if isinstance(self.grid[left_grid_row][left_grid_col], Piece):
```

Comment:
It's nice to use _isinstance_ here, but line 80 _if_ statement is redundant, you can use one of them as if and other as else instance.